config/data/langs.toml

```diff
@@ -182,21 +182,19 @@ Next
 args       = [ '/usr/bin/befunge', '-' ]
 experiment = 428
 size       = '2.81 MiB'
-version    = 'Befunge-98 1,001'
+version    = 'b7bf640'
```

Comment:
This is probably our way to go. Extra-terrestrial versioning schemes with the number of commits it took to drop their next version, and actually _use_ that number to define the version, that just benofunge.